tutor/templates/build/openedx/Dockerfile

```diff
@@ -207,6 +207,13 @@ RUN openedx-assets themes \
 # Create a data directory, which might be used (or not)
 RUN mkdir /openedx/data
 
+# If this "canary" file is missing from a container, then that indicates that a
+# local edx-platform was bind-mounted into that container, thus overwriting the
+# canary. This information is useful during edx-platform initialisation.
+RUN echo \
+  "This copy of edx-platform was built into a Docker image." \
+  > bindmount-canary
```

Comment:
:laughing: I wish I had a better solution, but I don't.

All I got is defining a custom template variable (via [Filters.ENV_TEMPLATE_VARIABLES](https://docs.tutor.overhang.io/reference/api/hooks/catalog.html#tutor.hooks.Filters.ENV_TEMPLATE_VARIABLES)) whenever edx-platform is bind-mounted. But it's more complex, not very intuitive, and I'm not even sure it would work, because of the order in which filters are applied...